examples/README.md

```diff
@@ -2,9 +2,11 @@
 
 ## Notebooks
 
-#### [`colab/text_classification_with_tf_hub_on_kaggle.ipynb`](colab/text_classification_with_tf_hub_on_kaggle.ipynb)
+### [`colab/`]
```

Comment:
@vasudevgupta7 can you fix this? Looks like you accidentally deleted the old first example